handler/ndnp.py

```diff
@@ -151,10 +151,13 @@ def __init__(self, batchfile):
 
         # iterate over the paths to the issues and create an item from each one
         for n, p in enumerate(self.paths):
-            print("Preprocessing item {0}/{1}...".format(n+1,
-                self.length), end='\r')
+            print("Preprocessing item {0}/{1}...".format(
+                n+1, self.length), end='\r'
+                )
             
-            # print(p)
+            if len(self.items) >= limit:
```

Comment:
@jwestgard 
I got the following error when I tried to load after this PR. 
```
  File "/apps/git/newspaper-batchload/handler/ndnp.py", line 158, in __init__
    if len(self.items) >= limit:
TypeError: unorderable types: int() >= NoneType()
```
Looks like this is happening because I did not specify a limit in the command line.

Should line 158 check for limit not null before comparing?
```
if limit and len(self.items) >= limit:
```

Also, can you please update the README with the usage/syntax of the limit?